salt/states/openvswitch_port.py

```diff
@@ -37,73 +37,79 @@ def present(name, bridge, type=None, id=None, remote=None, dst_port=None, intern
-    changes_port_created = {name: {'old': 'No port named {0} present.'.format(name),
-                                   'new': 'Created port {1} on bridge {0}.'.format(bridge, name),
-                                   }
-                            }
+    
+    comments = {}
 
-    comment_vlan_invalid_id = 'VLANs id must be between 0 and 4095.'
-    comment_vlan_invalid_name = 'Could not find network interface {0}.'.format(name)
-    comment_vlan_port_exists = 'Port {0} with access to VLAN {1} already exists on bridge {2}.'.format(name, id, bridge)
-    comment_vlan_created = 'Created port {0} with access to VLAN {1} on bridge {2}.'.format(name, id, bridge)
-    comment_vlan_notcreated = 'Unable to create port {0} with access to VLAN {1} on ' \
-                              'bridge {2}.'.format(name, id, bridge)
-    changes_vlan_created = {name: {'old': 'No port named {0} with access to VLAN {1} present on '
-                                          'bridge {2} present.'.format(name, id, bridge),
-                                   'new': 'Created port {1} with access to VLAN {2} on '
-                                          'bridge {0}.'.format(bridge, name, id),
+    comments['comment_bridge_notexists'] = 'Bridge {0} does not exist.'.format(bridge)
+    comments['comment_port_exists'] = 'Port {0} already exists.'.format(name)
+    comments['comment_port_created'] = 'Port {0} created on bridge {1}.'.format(name, bridge)
+    comments['comment_port_notcreated'] = 'Unable to create port {0} on bridge {1}.'.format(name, bridge)
+    comments['changes_port_created'] = {name: {'old': 'No port named {0} present.'.format(name),
+                                   'new': 'Created port {1} on bridge {0}.'.format(bridge, name),
                                    }
                             }
 
-    comment_gre_invalid_id = 'Id of GRE tunnel must be an unsigned 32-bit integer.'
-    comment_gre_interface_exists = 'GRE tunnel interface {0} with rempte ip {1} and key {2} ' \
-                                   'already exists on bridge {3}.'.format(name, remote, id, bridge)
-    comment_gre_created = 'Created GRE tunnel interface {0} with remote ip {1}  and key {2} ' \
-                          'on bridge {3}.'.format(name, remote, id, bridge)
-    comment_gre_notcreated = 'Unable to create GRE tunnel interface {0} with remote ip {1} and key {2} ' \
-                             'on bridge {3}.'.format(name, remote, id, bridge)
-    changes_gre_created = {name: {'old': 'No GRE tunnel interface {0} with remote ip {1} and key {2} '
-                                         'on bridge {3} present.'.format(name, remote, id, bridge),
-                                   'new': 'Created GRE tunnel interface {0} with remote ip {1} and key {2} '
-                                          'on bridge {3}.'.format(name, remote, id, bridge),
-                                   }
-                            }
+    if type:
```

Comment:
I agree with that, we could replace it with a variable named "tunnel_type".
However, I think this would break the state signature, as "type" comes from https://docs.saltstack.com/en/latest/ref/states/all/salt.states.openvswitch_port.html, doesn't it?